src/Uno.Toolkit.UI/Controls/LoadingView/LoadingView.cs

```diff
@@ -142,6 +142,9 @@ private void OnSourceChanged(DependencyPropertyChangedEventArgs e)
 
 			_subscription.Disposable = Source?.BindIsExecuting(UpdateVisualState);
 		}
+
+		private DataTemplate? loadingContentTemplatePlaceholder;
+		private DataTemplateSelector? loadingContentTemplateSelectorPlaceholder;
```

Comment:
Should those be reset if the original values change?

Reply:
yes, and there's probably a bit more refactoring to do. At this stage I just wanted to gauge whether or not there's a better approach before finalising this